src/style.css

```
@@ -0,0 +1,77 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+html {
+  font-weight: 300;
+  font-family: Avenir, Helvetica, sans-serif;
+  color: #3d4e61;
+}
+
+.bar {
+  width: 100%;
+  font-family: inherit;
+  font-weight: 400;
+  display: block;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.big__bar {
+  height: 70px;
+  margin: 20px 0;
+  padding-left: 62px;
+  background: url(./images/Search.svg) no-repeat;
+  background-size: 19px;
+  background-position: 26px;
+  font-size: 16px;
+  line-height: 21px;
+  color: #3d4e61;
+
+}
```

Don't leave empty lines in selectors
```suggestion
  color: #3d4e61;
}
```